test/Dataworker.loadData.ts

```diff
@@ -0,0 +1,301 @@
+    await updateAllClients();
+    const data3 = dataworkerInstance._loadData();
+    expect(data3.unfilledDeposits).to.deep.equal({
+      [destinationChainId]: [
+        { unfilledAmount: amountToDeposit.sub(fill1.fillAmount), data: deposit1 },
+        { unfilledAmount: amountToDeposit.mul(toBN(2)), data: deposit3 },
+      ],
+      [originChainId]: [
+        { unfilledAmount: amountToDeposit.sub(fill2.fillAmount), data: deposit2 },
+        { unfilledAmount: amountToDeposit.mul(toBN(2)), data: deposit4 },
+      ],
+    });
+
+    // One completely filled deposit per destination chain ID.
+    await buildFill(spokePool_2, erc20_2, depositor, depositor, relayer, deposit3, 1);
+    await buildFill(spokePool_1, erc20_1, depositor, depositor, relayer, deposit4, 1);
+    await updateAllClients();
+    const data4 = dataworkerInstance._loadData();
+    expect(data4.unfilledDeposits).to.deep.equal({
+      [destinationChainId]: [{ unfilledAmount: amountToDeposit.sub(fill1.fillAmount), data: deposit1 }],
+      [originChainId]: [{ unfilledAmount: amountToDeposit.sub(fill2.fillAmount), data: deposit2 }],
+    });
+
+    // All deposits are fulfilled
+    await buildFill(spokePool_2, erc20_2, depositor, depositor, relayer, deposit1, 1);
+    await buildFill(spokePool_1, erc20_1, depositor, depositor, relayer, deposit2, 1);
+    await updateAllClients();
+    const data5 = dataworkerInstance._loadData();
+    expect(data5.unfilledDeposits).to.deep.equal({});
+  });
+  it("Returns fills to refund", async function () {
+    await updateAllClients();
+
+    // Submit a valid fill
+    const deposit1 = await buildDeposit(
+      spokePool_1,
+      erc20_1,
+      depositor,
+      depositor,
+      destinationChainId,
+      amountToDeposit
+    );
+    const deposit2 = await buildDeposit(spokePool_2, erc20_2, depositor, depositor, originChainId, amountToDeposit);
+    const fill1 = await buildFill(spokePool_2, erc20_2, depositor, depositor, relayer, deposit1, 0.5);
+
+    // Should return one valid fill linked to the repayment chain ID
+    await updateAllClients();
+    const data1 = dataworkerInstance._loadData();
+    expect(data1.fillsToRefund).to.deep.equal({
+      [repaymentChainId]: { [relayer.address]: [fill1] },
+    });
+
+    // Submit two more fills: one for the same relayer and one for a different one.
+    const fill2 = await buildFill(spokePool_1, erc20_1, depositor, depositor, relayer, deposit2, 0.25);
+    const fill3 = await buildFill(spokePool_2, erc20_2, depositor, depositor, depositor, deposit1, 1);
+    await updateAllClients();
+    const data2 = dataworkerInstance._loadData();
+    expect(data2.fillsToRefund).to.deep.equal({
+      [repaymentChainId]: { [relayer.address]: [fill1, fill2], [depositor.address]: [fill3] },
+    });
+
+    // Submit fills without matching deposits. These should be ignored by the client.
+    // @dev: Switch the deposit data to make fills invalid.
```

Comment:
same nit as before around using the `@dev` syntax.
```suggestion
    // Switch the deposit data to make fills invalid.
```

Reply:
Fixed